keytree/src/lib.rs

```diff
@@ -211,18 +203,16 @@ mod tests {
         let xprv = Xprv::random(&mut rng);
         let xpub = xprv.to_xpub();
 
-        // the following are hard-coded based on the previous seed
-        let expected_dk = [
-            159, 7, 231, 190, 85, 81, 56, 122, 152, 186, 151, 124, 115, 45, 8, 13, 203, 15, 41,
-            160, 72, 227, 101, 105, 18, 198, 83, 62, 50, 238, 122, 237,
-        ];
-        let expected_compressed_point = CompressedRistretto::from_slice(&[
-            156, 102, 163, 57, 200, 52, 79, 146, 47, 195, 32, 108, 181, 218, 232, 20, 165, 148,
-            192, 23, 125, 211, 35, 92, 37, 77, 156, 64, 154, 101, 184, 8,
-        ]);
+        // hard-coded based on the previous seed
+        let compressed_pt_hex = "9c66a339c8344f922fc3206cb5dae814a594c0177dd3235c254d9c409a65b808";
+        let expected_compressed_point =
+            CompressedRistretto::from_slice(&hex::decode(compressed_pt_hex).unwrap());
```

Comment:
Likewise, I don't think I can avoid this decoding. (And, again, I'll inline `compressed_pt_hex`.)